docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -0,0 +1,229 @@
+```
+
+If you don't have the Azure DCAP client installed previously, please skip the content below.
+
+In most cases the Azure version of libdcap_quoteprov.so is located in /usr/lib. Check your path before changing. Here we use /usr/lib as an example.
+
+```bash
+sudo mv /usr/lib/libdcap_quoteprov.so /usr/lib/libdcap_quoteprov.so.azure
+```
+
+Otherwise the Azure one might still get used because $PATH might have /usr/lib before the path /usr/lib/x86_64-linux-gnu with the Intel version.
+
+### 3.3 Configure the qpl
+Edit the file /etc/sgx_default_qcnl.conf. To accept insecure HTTPS cert, set the option USE_SECURE_CERT to FALSE as we will use a local caching service which doesn't have a secure cert.
+
+```bash
+USE_SECURE_CERT=FALSE
+```
+
+Note:
+The cert mentioned in /etc/sgx_default_qcnl.conf is just for a regular TLS handshaking between the QPL and the PCCS. That cert itself has nothing to do with attestation process. It has no relationship with the certs (e.g., Provisioning Certification Key certs (PCK certs)) that being used in the attestation.
+
+Setting "USE_SECURE_CERT=FALSE" doesn't mean your attestation process is insecure. It just means QPL will accept a self-signed cert for TLS handshaking with PCCS. A CA-signed/self-signed cert might be a better word than a "secure/insecure" cert. But that's how exactly the /etc/sgx_default_qcnl.conf describes it. So we just document it accordingly.
+
+## 4 Set up local Provisioning Certificate Caching Service (PCCS)
+### 4.1 Register an Intel developer account and get a subscription key
+
+If you don't have an Intel account, go to https://www.intel.com to register one by using the button at the corner.
+
+![Intel Account](./images/intel-account.png)
+
+Sign in with your Intel account and then go to
+https://api.portal.trustedservices.intel.com/provisioning-certification
+
+You shall see the screen like this:
+
+![Subscribe1](./images/subscribe1.png)
+
+Click the subscribe button then you shall see a page like this.
+
+![Subscribe2](./images/subscribe2.png)
+
+Then click the subscribe button again. You shall see your subscription information.
+
+![Primary Key](./images/primary-key.png)
+
+Get the value of your primary key, which will be used during PCCS service bring up.
+
+### 4.2 Install PCCS by using its Debian package
+Install nodejs and npm if you haven't
+
+```bash
+curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash
+sudo apt-get install -y nodejs
+```
+
+From the [list of different versions of pccs](https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/web/sgx-dcap-pccs/), please download the most recent version that matches your OS version.
+
+In this document, we use sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb as an example.  Run the command below to download the package
+```bash
+cd ~
+wget https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/web/sgx-dcap-pccs/sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb
```

Comment:
Same as above. You can install this from intel's apt repo

Reply:
Thanks for the suggestions. I have added the content on installing pccs by directly using APT as a recommended approach.